scripts/mklfpk.bat

```diff
@@ -48,7 +48,17 @@ set FILENAME=%~3
 
 if not exist %DST% mkdir %DST%
 
-if not exist "%DST%\license.txt" (
+set is_exist=false
+if not exist "%DST%\win\" (
+	set is_exist=true
+	if not exist "%DST%\lib\" (
+		set is_exist=true
+	) else (
+		set is_exist=false
+	)
+)
+
+if "%is_exist%" == "true" (
```

Comment:
Will it make sense to flip true with false here?  
Variable name have opposite meaning

Reply:
After conversation with @dr-pain logic was changed. This variable was excluded.